.github/workflows/fabric-build.yml

```
@@ -20,19 +20,20 @@ jobs:
           restore-keys: |
             gradle-
       - name: checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: validate gradle wrapper
         uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.7)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)